test/fixtures/rules/css-modules/undefined.js

```diff
@@ -0,0 +1,15 @@
+// undefined.js
+import React from 'react';
+
+import styles from './test.css';
+
+// `css-modules/no-undef-class` - Alerts whenever a CSS class is not being used
+// ---------------------------------------------------------------------
+// Bad
+(function () {
+    const div = <div className={ styles.undefClass } />;
+})();
+// Good
+(function () {
+    const div = <div className={ [styles.usedClass, styles.unusedClass] } />;
```

Comment:
I don't like having `usedClass` and `unusedClass`. What if we simply name them `foo` and `bar`, and for the undefined one simply use `undef`?

Reply:
changed the names to .foo and .bar